backend/qemu.pm

```diff
@@ -570,6 +570,10 @@ sub start_qemu {
     my $num_networks = 1;
```

Comment:
What about:
```
my $num_networks = max($vars->{OFFLINE} ? 0 : 1, scalar @nicmac, scalar @nicvlan, scalar @tapdev);
```

Reply:
not sure this is an improvement

Reply:
For some reason it doesn't seem to work this way.